tools/mhm-tools/mhm_bankfull_discharge/bankfull_discharge.xml

```diff
@@ -0,0 +1,70 @@
+<tool id="bankfull_discharge" name="Calculate Bankfull Discharge and Wetted Perimeter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>
+        Calculate the river discharge at bankfull conditions and the bankfull width.
+    </description>
+    <requirements>
+        <requirement type="package" version="3.8">python</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python3 $__tool_directory__/_cli/__init__.py bankfull
+        -i "${input}" 
+        -o "${output}"
+        -r "${return_period}" 
+        -v "${var}"
+        #if $wetted_perimeter:
```

Comment:
You can just use `$wetter_perimeter` if you add `truevalue="--wetted_perimeter" falsevalue=""` to the xml.